Resources/Locale/en-US/prototypes/catalog/fills/crates/armory-crates.ftl

```diff
@@ -12,3 +12,6 @@ ent-CrateArmoryLaser = lasers crate
 
 ent-CrateArmoryPistols = pistols crate
     .desc = Contains two standard NT pistols with four mags. Requires Armory access to open.
+
+ent-CrateRevolver = revolver crate
+    .desc = Crate with one reliable Rex revolver, a revolver in case there is a threat for which a regular pistol will not be enough. Requires Armory access to open.
```

Comment:
revolver said twice sounds bad change the second to weapon or gun maybe
in case -> incase